### ISSUE TITLE
OrbitGenerator broken in tracking station

### ISSUE THREAD

**Issue by jrossignol** [jrossignol/ContractConfigurator#244]
From HarlyKin on [the forums](http://forum.kerbalspaceprogram.com/threads/101604-1-0-2-Contract-Configurator-v1-4-1-2015-06-18?p=2026555#post2026555):

> Getting an exception in my current game since updating. Shows in map view and tracking station
```
Code:
NullReferenceException: Object reference not set to an instance of an object
  at ContractConfigurator.Behaviour.OrbitGenerator.Draw () [0x00000] in <filename unknown>:0 
  at ContractConfigurator.Behaviour.OrbitGenerator+OrbitRenderer.OnPreCull () [0x00000] in <filename unknown>:0 
 ```
> Repeats a lot. Tried cancelling all contracts and using only ones that don't use orbit draw or waypoints but still showed up.

**Comment by lstanden** [comment]
:+1: Hitting this too... only noticed it because I'm exception hunting trying to get my install running smooth.

**Comment by jrossignol** [comment]
Fixed in 1.4.2 :)
